lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2544,9 +2555,36 @@ def test_calculate_report_csv_success(self, report_type, instructor_api_endpoint
         success_status = "Your {report_type} report is being generated! You can view the status of the generation task in the 'Pending Instructor Tasks' section.".format(report_type=report_type)
```

Comment:
Your {report_type} report is being created.
To view the status of the report, see Pending Instructor Tasks below.